### ISSUE TITLE
bem filter doesn't work when using -- or __

### ISSUE THREAD

**Issue by ramya-rao-a** [emmetio/expand-abbreviation#6]
`div.hello>.-there` expands to
```
<div class="hello">
        <div class="hello--there"></div>
    </div>
```

but
`div.hello>.--there` expands to
```
 <div class="hello">
        <div class="undefined--there"></div>
    </div>
```

**Comment by ramya-rao-a** [comment]
@sergeche Any updates on this?

We have cases like
`.vzr-st>.__content` being expected to expand to 
```
<div class="vzr-st">
            <div class="vzr-st__content"></div>
 </div>
```

instead it expands to 
```
<div class="vzr-st">
    <div class=" undefined_content"></div>
</div>
```

**Comment by sergeche** [comment]
Fixed in `@emmetio/html-transform@0.3.4`
